include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -831,19 +833,22 @@ class MGTwoLevelTransferNonNested<dim,
  *
  * The implementation of this class is explained in detail in @cite munch2022gc.
  */
-template <int dim, typename VectorType>
-class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
+template <int dim, typename Number>
+class MGTransferMF : public dealii::MGLevelGlobalTransfer<
```

Comment:
Is this really a good name for a public interface? I see where it comes from, but I personally do not like abbreviations because they just add to the mental load and reduce redundancy. I assume this is a temporary name, in which case this would be fine?